ui/app/src/modules/Preview/Tools/AssetsPanel.tsx

```diff
@@ -27,12 +44,104 @@ const translations = defineMessages({
   }
 });
 
+const initialSearchParameters = {
+  filters: {
+    'mime-type': ['image/png', 'image/jpeg']
+  }
+};
+
+const assetsPanelStyles = makeStyles(() => createStyles({
+  assetsPanelWrapper: {
+    padding: '15px'
+  },
+  search: {
+    marginBottom: '20px'
+  }
+}));
+
 export default function AssetsPanel() {
+  const classes = assetsPanelStyles({});
+  const activeSite = useActiveSiteId();
+  const [assets, setAssets] = useState(null);
+
+  setRequestForgeryToken();
+
+  useEffect(() => {
+    search(activeSite, initialSearchParameters).subscribe(({ items }) => {
+      setAssets(items);
+    })
+  }, [activeSite]);
+
   return (
     <ToolPanel title={translations.assetsPanel}>
-      <Typography component="h2" variant="subtitle1" style={{ padding: '10px' }}>
-        Assets Panel
-      </Typography>
+      <div className={classes.assetsPanelWrapper}>
+        <SearchBar onChange={() => {
+        }} keyword={''} classes={{ root: classes.search }}/>
+        {
+          assets?.map((item: SearchItem) =>
+            <AssetComponent name={item.name} key={item.path} path={item.path}/>
+          )
+        }
+      </div>
     </ToolPanel>
   );
 }
+
+const assetComponentStyles = makeStyles(() => createStyles({
+  card: {
+    cursor: 'move',
+    marginBottom: '10px',
+    '& .cardTitle': {
+      ...cardTitleStyles
+    },
+    '&:hover': {
+      backgroundColor: palette.gray.light1
+    },
+  },
+  avatar: {
+    color: palette.black
+  },
+  media: {
+    height: 0,
+    paddingTop: '56.25%'
+  },
+  action: {
+    marginTop: 0,
+    alignSelf: 'inherit'
+  },
+  root: {
+    padding: '0 16px',
+    height: '70px'
+  }
+}));
+
+function AssetComponent(props: any) {
```

Comment:
Use same card styles/structure from main search results page minus the checkbox and the options menu will have different options. Refactor search component as appropriately for card re-usage.